spec/requests/expanded_links_endpoint_spec.rb

```diff
@@ -1,125 +1,150 @@
-      content_id: edition.document.content_id,
-    )
-
-    FactoryGirl.create(:link,
-      link_set: link_set,
-      target_content_id: organisation.document.content_id,
-      link_type: "organisations",
-    )
-
-    get "/v2/expanded-links/#{edition.document.content_id}"
-
-    expect(parsed_response).to eql(
-      "version" => 0,
-      "content_id" => edition.document.content_id,
-      "expanded_links" => {
-        "organisations" => [
-          {
-            "analytics_identifier" => "GDS01",
-            "api_path" => "/api/content/my-super-org",
-            "base_path" => "/my-super-org",
-            "content_id" => "9b5ae6f5-f127-4843-9333-c157a404dd2d",
-            "schema_name" => "organisation",
-            "document_type" => "organisation",
-            "description" => "VAT rates for goods and services",
-            "locale" => "en",
-            "public_updated_at" => "2014-05-14T13:00:06Z",
-            "title" => "VAT rates",
-            "withdrawn" => false,
-            "links" => {},
-            "details" => { "body" => "<p>Something about VAT</p>\n" },
-          }
-        ],
-        "available_translations" => translations,
+        organisations: [{ content_id: SecureRandom.uuid }],
+        available_translations: [{ content_id: content_id }],
       }
-    )
+    end
+
+    before do
+      FactoryGirl.create(:expanded_links,
+        content_id: content_id,
+        locale: "en",
+        with_drafts: true,
+        expanded_links: expanded_links,
+        updated_at: updated_at,
+      )
+    end
+
+    it "is assumed to be 'en'" do
+      get "/v2/expanded-links/#{content_id}"
+
+      expect(parsed_response).to eql(
+        "generated" => updated_at.utc.iso8601,
+        "expanded_links" => expanded_links.as_json,
+      )
+    end
   end
 
-  it "returns only translations if there are no links" do
-    link_set = FactoryGirl.create(:link_set,
-      content_id: edition.document.content_id,
-    )
+  context "when requesting a different locale" do
+    let(:expanded_links) do
+      {
+        german_organisations: [{ content_id: SecureRandom.uuid }],
+        available_translations: [{ content_id: content_id }],
+      }
+    end
+
+    before do
+      FactoryGirl.create(:expanded_links,
+        content_id: content_id,
+        locale: "de",
+        with_drafts: true,
+        expanded_links: expanded_links,
+        updated_at: updated_at,
+      )
+    end
 
-    get "/v2/expanded-links/#{link_set.content_id}"
+    it "returns the links for that locale" do
+      get "/v2/expanded-links/#{content_id}", params: { locale: "de" }
 
-    expect(parsed_response).to eql(
-      "version" => 0,
-      "content_id" => edition.document.content_id,
-      "expanded_links" => {
-        "available_translations" => translations,
-      },
-    )
+      expect(parsed_response).to eql(
+        "generated" => updated_at.utc.iso8601,
+        "expanded_links" => expanded_links.as_json,
+      )
```

Comment:
We set the expanded_links in this context block so if it doesn't match that one the test will fail.